examples/asgilook/asgilook/cache.py

```diff
@@ -0,0 +1,55 @@
+import msgpack
+
+
+class RedisCache:
+    PREFIX = 'asgilook:'
+    INVALIDATE_ON = frozenset({'DELETE', 'POST', 'PUT'})
+    CACHE_HEADER = 'X-ASGILook-Cache'
+    TTL = 3600
+
+    def __init__(self, config):
+        self.config = config
+
+        # NOTE(vytas): To be initialized upon application startup (see the
+        #   method below).
+        self.redis = None
+
+    async def process_startup(self, scope, event):
+        if self.redis is None:
+            self.redis = await self.config.create_redis_pool(
+                self.config.redis_host)
+
+    async def serialize_response(self, resp):
```

Comment:
Should this be prefixed with an underscore? If nothing else, it would make it clear it is not part of the middleware interface.

Reply:
Right, I'll revise all attributes and methods, and privatize what I normally use to privatize.